cin_validator/ingress.py

```diff
@@ -9,7 +9,7 @@
 import pandas as pd
 
 # TODO make this work with from cin_validator.utils import get_values
-from .utils import get_values, make_census_period
+from .utils import get_values, make_census_period, process_date_columns
```

Comment:
Side note: the TODO above now works, you can replace ".utils" with "cin_validator.utils" and delete line 11.